run-benchmark.sh

```diff
@@ -67,12 +67,13 @@ fixPath() {
   echo $path
 }
 
-now() {
```

Comment:
i do not understand removing the function 'now' - it is useful to have a single location to format all date output strings. if there are 2-3 different formats desired, wrap each of them in a function (2-3 functions) and then the format desired is clear.  calling date with obstruse stings requires a person to read the man page if any options are not understood.